build/emoji/emoji_gen.mjs

```diff
@@ -7,38 +7,43 @@
 * 'mattermost-webapp/utils/emoji.jsx'
 * 'mattermost-webapp/sass/components/_emojisprite.scss'
 * 'mattermost-webapp/utils/emoji.json'
-* 'mattermost-server/model/emoji_data.go', (if SERVER_DIR environment variable is declared, otherwise it will be generated in 'mattermost-webapp/emoji_data.go')
+* 'mattermost-server/model/emoji_data.go', (if server-dir argument is passed with path to server, otherwise it will be generated in './emoji_data.go'")
 *
 * For help on how to use this script, run:
-* npm run gen-emoji -- --help
+* npm run make-emojis -- --help
 */
 
 import path from 'path';
 import * as Fs from 'fs/promises';
 import {readFileSync} from 'fs';
 
 import yargs from 'yargs';
-import jsonData from 'emoji-datasource/emoji.json';
-import jsonCategories from 'emoji-datasource/categories.json';
+import jsonData from 'emoji-datasource/emoji.json' assert {type: 'json'};
```

Comment:
nice job with this `assert {type: 'json'}` but is this out of box supported in node version we are using? i see it is still stage 3